wdl/transforms/draft3/src/test/scala/AstToWorkflowDefinitionElementSpec.scala

```diff
@@ -0,0 +1,97 @@
+import cats.data.Validated._
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers._
+import wdl.model.draft3.elements.ExpressionElement._
+import wdl.model.draft3.elements.{InputDeclarationElement, InputsSectionElement, IntermediateValueDeclarationElement, OutputDeclarationElement, OutputsSectionElement, PrimitiveTypeElement}
+import wom.types.{WomSingleFileType, WomStringType}
+import wdl.transforms.base.ast2wdlom.AstToWorkflowDefinitionElement._
+
+
+class AstToWorkflowDefinitionElementSpec extends AnyFlatSpec{
+  behavior of "Check Stdouts and Stderrs"
+
+  val mockInputSectionStdout = Option(InputsSectionElement(Vector(InputDeclarationElement(PrimitiveTypeElement(WomSingleFileType), "i", Some(StdoutElement)))))
+  val mockInputSectionStderr= Option(InputsSectionElement(Vector(InputDeclarationElement(PrimitiveTypeElement(WomSingleFileType), "i", Some(StderrElement)))))
+  val mockInputSectionNonStd= Option(InputsSectionElement(Vector(InputDeclarationElement(PrimitiveTypeElement(WomStringType), "more", Some(StringLiteral("more"))))))
```

Comment:
For consistent spacing
```suggestion
  val mockInputSectionStderr = Option(InputsSectionElement(Vector(InputDeclarationElement(PrimitiveTypeElement(WomSingleFileType), "i", Some(StderrElement)))))
  val mockInputSectionNonStd = Option(InputsSectionElement(Vector(InputDeclarationElement(PrimitiveTypeElement(WomStringType), "more", Some(StringLiteral("more"))))))
```